assets/css/sensei-course-theme/layout.scss

```diff
@@ -7,25 +16,34 @@
 
 	&__header {
 		margin: 0;
-		height: 75px;
 		position: sticky;
-		top: 0;
+		top: var(--top-offset);
 		background-color: #fff;
 		z-index: 100;
-		body.admin-bar & {
-			top: 32px;
-		}
+
 	}
 
 	&__sidebar {
+		position: fixed;
+		top: calc( var(--header-height) + var(--top-offset) );
+
```

Comment:
Is this extra line on purpose?

Reply:
Not really :) Sometimes I like to separate sections, but it can end up being totally random after some changes. Cleaned it up a bit in 65eea28.